controllers/analytics.ts

```diff
@@ -0,0 +1,41 @@
+const log = require("loglevel");
+const asyncHandler = require("express-async-handler");
+const Sentry = require("@sentry/node");
```

Comment:
i think we should start using only imports here
```suggestion
import log  from "loglevel";
import asyncHandler from "express-async-handler";
import Sentry from "@sentry/node";
```